### ISSUE TITLE
Single-column layout doesn't cover page width

### ISSUE THREAD

**Issue by michitux** [xwikisas/xwiki-pro-macros#206]
**Steps to reproduce**

Create a page with the following content:

```
{{layout}}
{{layout-section ac:type="single"}}
{{layout-cell}}
{{info}}Some information{{/info}}
{{/layout-cell}}
{{/layout-section}}
{{/layout}}
```

**Expected result:**

The info box spans the whole page.

**Actual result:**

The info box is much smaller than the whole page.

![image](https://github.com/xwikisas/xwiki-pro-macros/assets/198317/e283a973-ef94-42fa-9aba-f8edfdc50fad)

**Comment by Farcasut** [comment]
Hello, @michitux! Can you specify on which XWiki version you've encountered this issue and if it was a clean instance? I tried to replicate it on XWiki 13.10 and 14.10.1 and in both cases, I got the expected result.
13.10
![image](https://github.com/xwikisas/xwiki-pro-macros/assets/92780090/0e56ce21-ce2b-4b93-9b52-8e1ebc07c05a)
14.10.1
![image](https://github.com/xwikisas/xwiki-pro-macros/assets/92780090/5bea26e3-0357-4fba-9ee9-2e2575ef704c)

**Comment by michitux** [comment]
I think this has been fixed in the meantime by 65aa0370606f0faed7eb432b724126d818ee533b, I cannot reproduce after upgrading the pro macros. However, I wonder if the whitespace left of the content is desired.

**Comment by snazare** [comment]
Tested on 15.10.10 pro macros 1.18.0 - indeed the box takes most screen but there is an odd empty space at the beginning of the info box. @Farcasut would you have the chance to fix this?
